components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -258,7 +259,14 @@ export default function (props: { project?: Project; isAdminDashboard?: boolean
                                                         alt={p.info.startedBy}
                                                     />
                                                 )}
-                                                Triggered {formatDate(p.info.startedAt)}
+                                                <Tooltip
+                                                    className="w-fit"
+                                                    content={`Triggered At ${dayjs(p.info.startedAt)
```

Comment:
**suggestion**: It's to skip the duplicate prefix inside the tooltip that could be used in the UI.

```suggestion
                                                    content={`${dayjs(p.info.startedAt)
```